evalml/automl/automl_search.py

```diff
@@ -1656,3 +1684,16 @@ def get_ensembler_input_pipelines(self, ensemble_pipeline_id):
         return self._results["pipeline_results"][ensemble_pipeline_id][
             "input_pipeline_ids"
         ]
+
+    def is_timing_input_correct(self, timing):
```

Comment:
we should raise a `ValueError` or at least a warning if the user provides an input we don't accept. I think I would go for a `ValueError` though! Be sure to add a test case too.